permissions/autogenerated/reference.md

```diff
@@ -1,4 +1,66 @@
 
+## Permission Table
```

Comment:
Is this duplicate block a tauri bug or caused by the rc -> stable update or was this a manual edit?

idk, can you try deleting it and regenerating it then?

Reply:
Maybe it was because of the version change in Tauri? Honestly, I don’t know. I didn’t touch anything that was autogenerated, but I’ve already committed it after regenerating